Source/ASDisplayNode+Layout.mm

```diff
@@ -211,7 +211,7 @@ @implementation ASDisplayNode (ASLayoutInternal)
  * @discussion The size of a root node is determined by each subnode. Calling invalidateSize will let the root node know
  * that the intrinsic size of the receiver node is no longer valid and a resizing of the root node needs to happen.
  */
-- (void)_setNeedsLayoutFromAbove
+- (void)_u_setNeedsLayoutFromAbove
 {
   as_activity_create_for_scope("Set needs layout from above");
   ASDisplayNodeAssertThreadAffinity(self);
```

Comment:
Nit: Let's add the thread ownership assertion (i.e `ASDisplayNodeAssertLockUnownedByCurrentThread(__instanceLock__)`). We may enable it by default in the future.